src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -100,9 +101,10 @@ private static Person createEditedPerson(ReadOnlyPerson personToEdit,
         Phone updatedPhone = editPersonDescriptor.getPhone().orElse(personToEdit.getPhone());
         Email updatedEmail = editPersonDescriptor.getEmail().orElse(personToEdit.getEmail());
         Address updatedAddress = editPersonDescriptor.getAddress().orElse(personToEdit.getAddress());
+        Note note = personToEdit.getNote();
```

Comment:
to be consisitent, should note be named updatedNote instead?

Reply:
Alright, shall change it!